autoPyTorch/data/tabular_feature_validator.py

```diff
@@ -32,49 +31,36 @@ def _create_column_transformer(
     Args:
         preprocessors (Dict[str, List[BaseEstimator]]):
             Dictionary containing list of numerical and categorical preprocessors.
-        numerical_columns (List[str]):
-            List of names of numerical columns
         categorical_columns (List[str]):
             List of names of categorical columns
 
     Returns:
         ColumnTransformer
     """
 
-    numerical_pipeline = 'drop'
-    categorical_pipeline = 'drop'
-    if len(numerical_columns) > 0:
-        numerical_pipeline = make_pipeline(*preprocessors['numerical'])
-    if len(categorical_columns) > 0:
```

Comment:
it is being checked before this function is called. see [here](https://github.com/automl/Auto-PyTorch/blob/4698386a91bd3ede75eec9258c4d2f73599cede6/autoPyTorch/data/tabular_feature_validator.py#L164). In case we dont have categorical columns no column transformer will be created.

Reply:
True that, I thought we were going to check like before.